### DISCUSSION TITLE
GetMyLocation not working

### DISCUSSION THREAD

**Discussion by gscaduto** [TerriaJS/terriajs#6650] [Q&A] [answered]
Hi,

the button for my location not working:
"Origin does not have permission to use Geolocation service"

how can i activate it?

Thanks
Gabriele.

**Comment by tephenavies** [comment, marked as answer]
Hi Gabriele. I think the issue you're coming up against is that geolocation is blocked by browsers when a site is not accessed over https. This includes http://localhost
